pages/login.vue

```diff
@@ -43,22 +43,10 @@ const submit = createSubmitHandler(schema, (values) => {
       <UButton
         type="submit"
         class="mt-20 mx-auto"
+        label="Zaloguj się"
```

Comment:
W sumie czemu tak?

Reply:
Bo skoro jest opcja by to przekazać jako parametr komponentu to w sumie czemu z tego nie korzystać?